src/tools/rbd/action/Group.cc

```diff
@@ -442,8 +447,14 @@ int execute_list_images(const po::variables_map &vm,
     }
   );
 
-  if (f)
+  if (f){
     f->open_array_section("images");
+    f->open_object_section("group");
+    f->dump_string("group_id", id);
```

Comment:
> I think we do not need add a new command, because the new command and `group image ls` have similar functions, just added `group id`

The tracker ticket talks about adding a new command for a reason: we don't want to "pollute" the image listing -- certainly not the way it's done here where `group_id` is dumped in the middle of `images` array.

Let's start with `rbd group info` command showing just the group ID in a format similar to `rbd info` command.  For example:
```
$ rbd group info mypool/mygroup
rbd group 'mygroup':
        id: 10636dcb04e5
```